mne/chpi.py

```diff
@@ -923,8 +999,7 @@ def compute_chpi_amplitudes(raw, t_step_min=0.01, t_window='auto',
     raw : instance of Raw
         Raw data with cHPI information.
     t_step_min : float
-        Minimum time step to use. If correlations are sufficiently high,
-        t_step_max will be used.
+        Minimum time step to use.
```

Comment:
Indeed the old description was a bit misleading, it's from when there was a single monolithic function. Now that head pos calculation is split into three separate steps, the stuff about `t_step_max` doesn't make much sense here